### DISCUSSION TITLE
Can you add supports for the GOBI 6000 WWAN Card?

### DISCUSSION THREAD

**Discussion by huynhhoangkha** [macOS-Dell-Latitude-E7470/macOS-Dell-Latitude-E7470#23] [General]
I have a GOBI 6000 - D5811e WWAN card on my E7470.
But I can't find the solution to using it on the Hackintosh.
Could you please help with this?

**Comment by adityabakare** [comment]
I'm sorry but I can't. I don't have the hardware to test, and I don't even know if WWAN cards still work on macOS.
